src/EnergyPlus/ScheduleManager.hh

```diff
@@ -86,11 +81,13 @@ namespace ScheduleManager {
         Num
     };
 
-    // DERIVED TYPE DEFINITIONS
-
-    // INTERFACE BLOCK SPECIFICATIONS
-
-    // MODULE VARIABLE DECLARATIONS:
+    enum class OutputScheduleReportLevel
```

Comment:
Just throwing this out there. The word `Schedule` should not appear in `enum class` names that are in the `Schedule`, because the fully resolved name will have `Schedule` twice. This way, it's shorter and helps us get rid of `using`. Something to think about.